### PR TITLE
Fix ZHA device removal navigation

### PR DESCRIPTION
**Pull request by dmulcahey**
## Proposed change

Currently when you remove a device the device page flips to a "device not found" page. This PR causes the UI to navigate to the ZHA devices list post device removal.

## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/devices/device-detail/integration-elements/zha/ha-device-actions-zha.ts [hunk 1]
```diff
@@ -119,9 +122,18 @@ export class HaDeviceActionsZha extends LitElement {
...
+        ieee_address: this._zhaDevice!.ieee,
+      })
+      .then(() => {
+        navigate(
+          this,
+          "/config/devices/dashboard?historyBack=1&config_entry=" +
+            this.configEntry.entry_id,
+          true
+        );
+      });
```
Comment (bramkragten) [inline src/panels/config/devices/device-detail/integration-elements/zha/ha-device-actions-zha.ts:136]:
```suggestion
    await this.hass
      .callService("zha", "remove", {
        ieee_address: this._zhaDevice!.ieee,
      });
    navigate(
      this,
      "/config/devices/dashboard?historyBack=1&config_entry=" +
        this.configEntry.entry_id,
      true
    );
```

Comment (bramkragten) [reply src/panels/config/devices/device-detail/integration-elements/zha/ha-device-actions-zha.ts:136]: Can't we just navigate back?
`history.back();`

Comment (dmulcahey) [reply src/panels/config/devices/device-detail/integration-elements/zha/ha-device-actions-zha.ts:136]: yep. Didn't think of that :(

Comment (dmulcahey) [reply src/panels/config/devices/device-detail/integration-elements/zha/ha-device-actions-zha.ts:136]: fixed here: #6638

**Comment by dmulcahey** [comment]
Opened a new PR as: #6638

**Pull request closed without merging**